src/main/java/org/opensearch/knn/index/util/Lucene.java

```diff
@@ -45,16 +48,25 @@ public class Lucene extends JVMLibrary {
         ).addSpaces(SpaceType.L2, SpaceType.COSINESIMIL).build()
     );
 
-    final static Lucene INSTANCE = new Lucene(METHODS, Version.LATEST.toString());
+    private final static Map<SpaceType, Function<Float, Float>> DISTANCE_TRANSLATIONS = ImmutableMap.<
+        SpaceType,
+        Function<Float, Float>>builder()
+        .put(SpaceType.COSINESIMIL, distance -> (2 - distance) / 2)
+        .put(SpaceType.L2, distance -> 1 / (1 + distance))
```

Comment:
For this, just to re-comfirm, customers need to input distance as euclidean^2 correct?

Reply:
Yes, user need provide distance input as euclidean^2